src/screens/ContactScreen/index.js

```diff
@@ -383,7 +449,7 @@ const ContactScreen = ({navigation}) => {
         </View>
       )}
 
-      {isRecyclerViewShown && !isSearchMode && (
+      {isRecyclerViewShown && usersSearch.length <= 0 && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for showing the RecyclerView has been updated to check `usersSearch.length <= 0`. This is a sensible change as it ensures that the RecyclerView is only shown when there are no search results.



<!-- This is an auto-generated comment by OSS CodeRabbit -->